src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -222,6 +223,13 @@ protected override void AddResponseFileCommands(CommandLineBuilderExtension comm
             commandLine.AppendSwitchIfNotNull("/nullable:", Nullable);
             commandLine.AppendWhenTrue("/nosdkpath", _store, nameof(DisableSdkPath));
```

Comment:
The way this is organized now if a customer has `<DisableSdkPath>true</DisableSdkPath>` then we will respect that option in `dotnet build` but not `msbuild`. Believe that before this change this option always overrode the implicit sdk path that was provided. Is that the case?